README.md

```diff
@@ -31,6 +31,10 @@ Version marked as 2.0.x are compatible with SilverStripe 3.1, with 2.0.1 compati
 * [SilverStripe 2.4 compatible version](https://github.com/silverstripe-australia/silverstripe-multivaluefield/tree/ss24)
 
 
+## Installation
+
+	composer require silverstripe/multivaluefield
```

Comment:
I realise this is a bit old now, but two suggestions:

* Use three backticks for code blocks rather than indenting it
* The module name has changed to symbiote/silverstripe-multivaluefield

If you could update these two things then we can merge